docs/api.md

```diff
@@ -82,6 +84,10 @@ For possible values of `opts.tracker` see the
 For possible values of `opts.blocklist` see the
 [`load-ip-set` documentation](https://github.com/webtorrent/load-ip-set#usage).
 
+For `opts.natUpnp` and `opts.natPmp`, if both are set to `true`, PMP will be attempted first, then fallback to UPNP. 
```

Comment:
NodeJS only.

Reply:
The docs have been amended to note this is NodeJS only.